linesearch.go

```diff
@@ -63,46 +60,34 @@ func (ls *LinesearchMethod) Iterate(loc *Location) (Operation, error) {
 		// Init was not called. What to do? What about ls's internal state?
```

Comment:
Presumably this can then never happen, and could panic, unless we think there is a valid reason for a Linesearcher to return NoOperation.